assets/templates/partials/release/contents/about-the-data.tmpl

```diff
@@ -38,17 +35,14 @@
         </a>
       </h2>
       <p>
-        These statistics have been given National Statistics status by the
-        Office for Statistics Regulation. They have been produced by following
-        the standards set out in the <a href="https://code.statisticsauthority.gov.uk/the-code/">Code of Practice for Statistics</a>.
-        They meet the highest standards of trustworthiness, quality and public
-        value. This means that they:
+        {{- localise "NationalStatisticsBody1" $.Language 1 -}} <a href="https://code.statisticsauthority.gov.uk/the-code/">{{- localise "NationalStatisticsBodyLink" $.Language 1 -}}</a>.
```

Comment:
- Prefix all locale keys with `AboutData` for consistency with above changes
- Avoid long lines for the sake of readability:

```
{{- localise "NationalStatisticsBody1" $.Language 1 -}}
<a href="https://code.statisticsauthority.gov.uk/the-code/">
  {{- localise "NationalStatisticsBodyLink" $.Language 1 -}}
</a>.
```